cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -211,260 +223,412 @@ class OperandNode extends Node, TOperandNode {
 
   override IRType getType() { result = op.getIRType() }
 
-  override Location getLocation() { result = op.getLocation() }
+  final override Location getLocationImpl() { result = op.getLocation() }
 
-  override string toString() { result = this.getOperand().toString() }
+  override string toStringImpl() { result = this.getOperand().toString() }
 }
 
 /**
  * INTERNAL: do not use.
```

Comment:
It probably could, yeah. It's a bit annoying since that would imply removing the private annotation on the cached module `Cached` that wraps the IPA types.

Reply:
Couldn't that also go in DataFlowPrivate?

Reply:
Hmmm... Good question. Maybe it could, yeah.